elo_boat.py

```diff
@@ -249,15 +246,15 @@ async def allstats(ctx):
             total_creepkill = row[7]
             total_bounty = row[8]
             total_bountyfeed = row[9]
-            total_goldgathered = row[10]
+            # total_goldgathered = row[10]
             # total_dodosfound = row[11]
             total_chatcounter = row[12]
             # total_kickcounter = row[13]
             KD = round(total_kills / total_death, 2)
             # mean_assist = round(total_assist / total_games, 2)

```

Comment:
revert all the commented variables, we better have working variables than commented lines of codes